v19.1/contribute-to-cockroachdb.md

```diff
@@ -81,7 +81,7 @@ That being said, if you’re up for the challenge there are a few prerequisites.
 
 1. Convince the CockroachDB team of your ability to drive a larger project. For this, you can either work with the CockroachDB team on mid-sized projects first, or provide some other evidence that you are an expert at building high quality distributed systems.
     {{site.data.alerts.callout_info}}The engineering team may not support further exploration if they think the project is too complex for an external contributor or if we already have a schedule for this work and you cannot guarantee delivery on that schedule.{{site.data.alerts.end}}
-2. Make a high-level proposal for your project, declaring your intention, the area of functionality, and how much time (and, if applicable, people) you have available to work on the project. You should do this via a GitHub issue.
+2. Make a high-level proposal for your project, declaring your intention, the area of functionality, and how much time (and, if applicable, people) you have available to work on the project. You should do this via a [GitHub issue](https://github.com/cockroachdb/docs/issues/new?labels=community).
```

Comment:
This is a good idea, but you need to link to the cockroach repo, not the docs repo. I'll make this change in a separate PR.